docs/source/_static/depthai_env_setup.sh

```diff
@@ -177,3 +199,127 @@ else
     echo "ERROR: Host not supported"
     exit 99
 fi
+
+echo "Finished installing global libraries."
+
+# add depthai working dir to .bashrc if its not already there
+COMMENT='# Home directory of depthai and depthai app, enables to run <run_demo> in terminal'
+BASHRC="$HOME/.bashrc"
+ADD_DEPTHAI_TO_PATH='export PATH=$PATH'":$WORKING_DIR"
+ADD_LUXONIS_DEPTHAI_HOME='export LUXONIS_DEPTHAI_HOME='"$WORKING_DIR"
+
+# add to .bashrc only if it is not in there already
+write_in_file "$COMMENT" "$BASHRC"
+write_in_file "$ADD_DEPTHAI_TO_PATH" "$BASHRC"
+write_in_file "$ADD_LUXONIS_DEPTHAI_HOME" "$BASHRC"
+
+
+if [[ $(uname -s) == "Darwin" ]]; then
+  echo _____________________________
+  echo "Calling macOS_installer.sh"
+  echo _____________________________
+  echo "Running macOS installer."
+  echo "Upgrading brew."
+  brew upgrade
+
+  # clone depthai form git
+  if [ -d "$WORKING_DIR" ]; then
+     echo "Demo app already downloaded. Checking out main and updating."
+  else
+     echo "Downloading demo app."
+     git clone https://github.com/luxonis/depthai.git "$WORKING_DIR"
+  fi
+  cd "$WORKING_DIR"
+  git fetch
+  git checkout demo_app_installation_v2
+  git pull
+
+  # install python 3.10 and python dependencies
+  brew update
+  echo "installing python 3.10"
+  brew install python@3.10
```

Comment:
Same as for Linux - I'd install python only if current python isn't adequate (or isn't available)
We've got broad Python version support, so IMO no need to pin down to one specific version